Expensify/App
# Fix text selection on chats

Is this good? I kind of can't believe it is working, but it seems so?

### Fixed Issues
Fixes https://github.com/Expensify/Expensify/issues/142340

### Tests (both ios and web)
- Open the app, check when you switch to a chat you are in the latest message
- Select text from chat, check it's selected and copied in order:
